test/functional/conftest.py

```diff
@@ -387,6 +388,15 @@ def supports_gce():
     return _env_vars_exposed(env_vars) and HAS_GOOGLE_AUTH
 
 
+@pytest.helpers.register
+def supports_hetznercloud():
+    pytest.importorskip('hcloud')  # Ansible provides no import
```

Comment:
I don't understand this comment.
```suggestion
    pytest.importorskip('hcloud')
```

Reply:
All other functions try to import driver dependencies from Ansible, this does not. This comment will stop others wondering why we did not follow the same approach and trying to go luck at Ansible modules ...

Reply:
Suggest an improvement but let's not delete this context.

Reply:
When I read this I don't see any other approaches. Plz put the explanation there